js/exporter/exceljs/export.js

```diff
@@ -247,15 +268,26 @@ export const Export = {
                 this.setAlignment(excelCell, wrapText, horizontalAlignment);
             }
 
-            if(isFunction(customizeCell)) {
-                customizeCell(privateOptions._getCustomizeCellOptions(excelCell, cell));
-            }
-
             if(privateOptions._needMergeRange(rowIndex, headerRowCount)) {
-                const mergeRange = this.tryGetMergeRange(rowIndex, cellIndex, mergedCells, dataProvider);
-                if(isDefined(mergeRange)) {
-                    mergeRanges.push(mergeRange);
+                if(this.cellInMergeRanges(mergedCells, rowIndex, cellIndex)) {
+                    const cell = mergedCells[rowIndex][cellIndex];
+                    if(cell.merged === false) {
+                        excelCell.style = cell.masterCell.style;
+                        excelCell.value = cell.masterCell.value;
+                    }
+                } else {
+                    const mergeRange = this.tryGetMergeRange(excelCell, mergedCells, rowIndex, cellIndex, dataProvider, mergeRowFieldValues, mergeColumnFieldValues, privateOptions);
+
+                    if(isDefined(mergeRange)) {
+                        mergeRanges.push(mergeRange);
+                    }
                 }
+
+
+            }
+
+            if(isFunction(customizeCell)) {
+                customizeCell(privateOptions._getCustomizeCellOptions(excelCell, cellData.cellSourceData));
```

Comment:
will we pass the 'this cell is a part of a merge range' into the customizeCell handler?

Reply:
I think it's not require. The customer knows 'this cell is a part of a merge range' in source pivotGrid table. There is the property 'master' in pivotCell object with adress of 'master' cell.